terminal_api_flows/__init__.py

```diff
@@ -44,9 +44,12 @@ def http_request(endpoint, action, json_body="") -> (urllib3.HTTPResponse, dict)
         },
         body=json_body
     )
-    # print("---------------------------")
-    # print(http_request.data)
-    # print("---------------------------")
+    try:
+        print("---------------------------")
+        print(http_request.data)
+        print("---------------------------")
+    except:
```

Comment:
As I understand we use those prints for development debug?

If yes - Should we comment it?

if no - better write `if statement`
```
if http_request:
    print("---------------------------")
    print(http_request.data)
    print("---------------------------")
```